vnext/ReactUWP/Modules/AppThemeModuleUwp.cpp

```diff
@@ -50,12 +62,44 @@ const std::string AppTheme::getCurrentTheme()
   return m_currentTheme == winrt::ApplicationTheme::Light ? AppTheme::light : AppTheme::dark;
 }
 
+bool AppTheme::getIsHighContrast()
+{
+  return m_accessibilitySettings.HighContrast();;
```

Comment:
>; [](start = 48, length = 1)

nit:  extra semicolon

Reply:
Shoot, merged before the page fully loaded. Will fix this on next PR.